settings.coffee

```diff
@@ -148,6 +148,8 @@ settings =
 		compilesDir:  Path.join(DATA_DIR, "compiles")
 		# Where to cache downloaded URLs for the CLSI
 		clsiCacheDir: Path.join(DATA_DIR, "cache")
+		# Where to write the project to disk before running LaTeX on it
```

Comment:
This new directory separates project files from generated files and contains `output.XYZ` (`output.pdf`/`output.log`/...) files only for downloading by the user. WDYT about this version of the comment?

```suggestion
		# Where to write the output files to disk after running LaTeX
```

Reply:
Good catch :tada: , a copy-paste mistake on my part